shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -52,16 +52,15 @@ IOSRenderingAPI GetRenderingAPIForProcess(bool force_software) {
 #if TARGET_OS_SIMULATOR
```

Comment:
The comment right above this (which GitHub isn't letting me comment on) still references OpenGL

Reply:
Reworded the comment a bit.